internal/lm/nvml.go

```diff
@@ -254,3 +356,20 @@ func getDeviceClasses(devices []resource.Device) ([]uint32, error) {
 	}
 	return classes, nil
 }
+
+func generateUUIDs(seed string) string {
```

Comment:
```suggestion
func generateUUID(seed string) string {
```

Reply:
Also, does it make sense to just use an existing hash function instead of rolling our own?

https://yourbasic.org/golang/hash-md5-sha256-string-file/